cmd/lenses-cli/export_command.go

```diff
@@ -619,7 +619,9 @@ func writeSchemas(cmd *cobra.Command) error {
 			continue
 		}
 
-		return writeSchema(cmd, subject, 0)
+		if err := writeSchema(cmd, subject, 0); err != nil {
+			golog.Error(fmt.Sprintf("Error while exporting schema [%s]", subject))
```

Comment:
you miss `return err` the error in the end. otherwise it's not idiomatic go. Returning `nil` when something happened in a method which needs the error back to probably some check will cause a problem.

Reply:
I see what you are saying, although in the context of the cli app I think that it makes sense to just handle the error by printing it in the console and not having it bubble up in the stack

Reply:
In generally this is not the effective error handling for Go. Printing is fine and it’s good to have a message for the error. But returning back a `nil` when there is an error for a typical Go developer means there is no issue which can cause lot of troubles.

The typical go developer will check if this method hasn't an error, if it has it should do something. Also, `cobra` does this, that's why asks for returning an error. More specifically `cobra.Command.RunE` expects to see an error if there's something bad and now we break the contract.